lib/tasks/world_taxon_update.rake

```diff
@@ -0,0 +1,156 @@
+# This task is to update the titles of each worldwide taxon to include the country name they relate to.
+# This work is necessary to meet the following WCAG guidelines:
+# https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html
+# https://www.w3.org/WAI/WCAG22/Techniques/failures/F25
+# For accessibility reasons, taxon titles need to be more unique. Having multiple versions of pages for different
+# countries all titled e.g. 'Trade and invest' goes against WCAG guidelines.
+
+ROOT_CONTENT_ID = "91b8ef20-74e7-4552-880c-50e6d73c2ff9".freeze
+
+namespace :worldwide do
+  desc "Update worldwide taxon titles (external name) to include the name of the country they relate to"
+
+  task :add_country_name_to_title, %i[log_file_path] => :environment do |_, args|
+    log_file = nil
+    if args[:log_file_path]
+      log_file = File.open(args[:log_file_path], "w")
+      log_file.puts("Updating each worldwide taxon to include country name in their title")
+    end
+    puts("Updating each worldwide taxon to include country name in their title")
+
+    total_taxon_updates = 0
+
+    # Build a taxonomy tree with the grandparent
+    # (common ancestor e.g. /world/all - Help and services around the world) as the root
+    taxonomy = Taxonomy::ExpandedTaxonomy.new(ROOT_CONTENT_ID).build.child_expansion
+    log_file&.puts("Taxonomy has size #{taxonomy.tree.size}")
+    puts("Taxonomy has size #{taxonomy.tree.size}")
+
+    taxonomy.tree.each do |linked_item|
+      # Tree includes root (world/all) - skip that or if the taxon is a GENERIC (template) version
+      next if linked_item.content_id == ROOT_CONTENT_ID || linked_item.internal_name.include?("(GENERIC)")
+
+      internal_name = linked_item.internal_name
+      message = "Internal name = #{internal_name}"
+      log_file ? log_file.puts(message) : puts(message)
+
+      # As this is a tree, we reach all grandchildren without another loop (not a nested array)
+      # example grandchild url /world/passports-and-emergency-travel-documents-cape-verde
+
+      # Skip titles where the country name is already included at the end
+      # e.g. If child - country pages (parent e.g. /world/argentina - UK help and services in Argentina)
+      if internal_name.start_with?("UK help and services in ") || internal_name.start_with?("Living in") || internal_name.start_with?("Travelling to")
+        message = "Skipping #{internal_name} as it already includes the country name"
+        log_file ? log_file.puts(message) : puts(message)
+        next
+      end
```

Comment:
Could we take lines 31, and 42-45 into one separate conditional method we can call in line 31, something like:
 `next if skip_item?(linked_item)`. We will only need to remember to use `linked_item.internal_name` as `internal_name` will be defined later